source/nodejs/adaptivecards-designer-app/src/app.ts

```diff
@@ -26,6 +26,16 @@ window.onload = function() {
     ACDesigner.Strings.toolboxes.propertySheet.title = "Custom title";
     ACDesigner.Strings.toolboxes.sampleDataEditor.title = "Custom title";
     ACDesigner.Strings.toolboxes.toolPalette.title = "Custom title";
+    */
+
+	// Uncomment to configure pic2card service
+	/*
+	ACDesigner.Pic2Card.pic2cardService = "https://<<your-pic2Card-service-endpoint>>";
+    */
+
+    // To Configure path for pic2card image usage policy
```

Comment:
nit: spacing/tabs seems a bit off on these lines

Reply:
This one (line:33), on the diff viewer it shows extra space. On actual file it seems correct only.